notifications/alliance_selections.py

```diff
@@ -9,8 +9,6 @@ class AllianceSelectionNotification(BaseNotification):
 
     def __init__(self, event):
         self.event = event
-        self._event_feed = event.key_name
```

Comment:
The `_event_feed` and `_district_feed` don't look to be used anymore - I removed them from all the models just to keep things simple